### DISCUSSION TITLE
Impossible to add users

### DISCUSSION THREAD

**Discussion by sevyb** [craigk5n/webcalendar#406] [Q&A]
Hi there. I have the latest version of Webcalendar 1.9.6 but cannot add users. When I open the user management page, I have a wheel turning endlessly where I should view the users list. I get the popup to add new users, but when I try to save a new user, the system tells me that there is an internal server error. Any idea, what could help? Thanks in advance. Yves

**Comment by craigk5n** [comment]
Suggestions:
- Try version 1.9.8
- Check the Apache error log (commonly `/var/log/apache2/error.log` on Linux).  If there is something there (typical for a 500 error), post it to this discussion.

**Reply by sevyb** [reply]
Thanks, that helped!
